consensus/processor.go

```diff
@@ -95,9 +99,16 @@ func NewConfiguredProcessor(validator SignedMessageValidator, rewarder BlockRewa
 func (p *DefaultProcessor) ProcessBlock(ctx context.Context, st state.Tree, vms vm.StorageMap, blk *types.Block, ancestors []types.TipSet) ([]*ApplicationResult, error) {
 	var emptyResults []*ApplicationResult
 
+	ctx, err := tag.New(ctx, tag.Insert(metrics.KeyMethod, "ProcessBlock"))
```

Comment:
Blocking: Errors here are very unfortunate, but we should not wedge the node due to a metrics failure. Log an ERROR instead (or better, have the error happen much sooner like at node initialisation). I suggest factoring the 4-line block out to a simple helper function in `metrics` to reduce noise and document what it's doing.

Non-blocking, but could you confirm whether this is the best practise way of defining the metrics? I would have expected it to happen at construction, in a factory method, or possibly statically (or DI) rather than at point of use.

Edit: in light of metrics/constants.go I'm not sure what this is doing at all, actually.

Reply:
Tags are used in a few of the opencensus [examples](https://github.com/census-instrumentation/opencensus-go/tree/master/examples), and according to the [opencensus spec](https://github.com/census-instrumentation/opencensus-specs/blob/master/tags/TagMap.md) tags are:
 >used to label anything that is associated with a specific operation, such as an HTTP request.

I think adding tags is is more reduant than helpful since we already get the method name via the view -- I will remove them.